python/jittor/test/test_grad.py

```diff
@@ -151,7 +151,16 @@ def test_nth_grad(self):
         self.assertEqual(dx.data, 4*2**3)
         self.assertEqual(ddx.data, 4*3*2**2)
         self.assertEqual(dddx.data, 4*3*2*2**1)
-        
+
+    def test_no_grad(self):
+        a = jt.array(1.0)
+        with jt.no_grad():
+            b = a
+            for i in range(10):
+                b = b.clone() + 1
+        assert b.data == 11
+        jt.clean()
+        assert jt.liveness_info()["lived_vars"] == 2
```

Comment:
why this test code can test no_grad scope? Not need to log some info about grad? 0-0 why jt.liveness_info()["lived_vars"] is 2?

Reply:
no_grad scope was added in this PR too, because the grad is stoped, so the variable between a and b can be released, this makes the `lived_vars` becomes 2(a and b)